### DISCUSSION TITLE
Home Assistant OS 11.0

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2810] [Announcements]
Home Assistant OS 11.0 is built from Buildroot version 2023.02.05. With this release Home Assistant OS uses Docker 24.0.6 and containerd 1.7.6.

Most notable changes are:
  * :warning: Breaking Change: On Arm based boards network names are enumerated based on device tree. This means that the first Ethernet devices will no longer be named `eth0` but `end0`. The network configuration is automatically applied to the new network interface.
  * Using Linux' Preemptible Kernel configuration across all boards (lower latencies even on busy systems)
  * Filesystem freeze is relayed to database engines on Proxmox/KVM based VMs (this makes VM snapshots always coherent)
  * Various Bluetooth improvements (newer BlueZ, better fix Bluetooth LE advertisement stalls, Bluetooth device cache management)
  * Improved Kernel configuration should improve Docker's overlayfs performance
  * Improved containerd build and configuration makes containerd use less memory and CPU resources
  * Adjusted development work-flow which allows more incremental changes

---

Changes between 11.0.rc2 and 11.0

## Home Assistant Operating System

* Bump buildroot to update libcurl to v8.4.0 (#2804) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.0'>Home Assistant OS 11.0</a>.</em>

**Comment by VNRARA** [comment]
> Filesystem freeze is relayed to database engines on Proxmox/KVM based VMs (this makes VM snapshots always coherent)

Does this mean it HAOS can handle my host (DS220+) making a snapshot without freaking out about not being able to access the disk now?

**Reply by agners** [reply]
I am not familiar with the Synology hypervisor. It seems that it is branded as Virtual Machine Manager (VMM). But it's unclear if it is based on KVM, or supports the same interfaces, so I can't really tell. It does seem to me that they built their own hypervisor and/or heavily modified QEMU/KVM. So it is unlikely that this works in practice :cry:

**Comment by Doughboy68** [comment]
Notice now in Proxmox significantly less Memory usage in the VM summary.

**Comment by Impact123** [comment]
Best to check that in the VM itself with something like `free -h`/`glances`. PVE's memory statistics aren't really accurate in this case as they also count buffers and cache.

**Reply by agners** [reply]
Agreed!

Don't expect huge memory savings. There are some binaries smaller (particular containerd), which should lead to slightly less memory usage, but this is probably in the tens of megabytes range.

Often memory usage over time increases, so it is hard to compare.

**Comment by mortalone** [comment]
wow. what a speed/performance/latency boost.

VERY noticable on my system. very nice indeed :)
